.github/workflows/job-cmakebuild-macOS.yml

```diff
@@ -29,10 +29,10 @@ jobs:
         with:
           name: Build-macOS-${{ inputs.architecture }}-${{inputs.configuration}}
           path: |
-            Result.Darwin.x64.${{inputs.configuration}}/Panzerfaust/${{inputs.configuration}}/net6.0/osx-${{ inputs.architecture }}/publish/
```

Comment:
As for the script we should do the same for pipeline files

I think we should expose it at Engine-CI.yml file as strategy and then push it down to needed jobs